mdakits/menger_curvature/metadata.yaml

```
@@ -0,0 +1,77 @@
+project_name: Menger_Curvature
+
+authors:
+  - https://github.com/EtienneReboul/menger_curvature/blob/main/AUTHORS.md
+
+
+maintainers:
+  - IBPCREBOUL
+  - EtienneReboul
+
+description:
+    This project aims to provide a simple MDAkit for JIT accelerated Menger curvature calculation.
+    The idea is to associate a value of curvature to as many residues as possible in a polymer. 
+    If one has access to several conformations , the average value of the curvature (LC) and its standard deviation (LF) 
+    are valuable information to characterize the local dynamics of the backbone.
+
+
+keywords:
+  - IDP
+  - IDR
+  - flexibility
+  - Menger curvature
+  - protein
+
+license: GPL-2.0-or-later
+
+
+project_home: https://github.com/EtienneReboul/menger_curvature
+
+
+documentation_home: https://menger-curvature.readthedocs.io/
+
+
+documentation_type: UserGuide + API
+
+
+src_install:
+  - pip install git+https://github.com/EtienneReboul/menger_curvature@main
+
+
+import_name: menger
+
+
+python_requires: ">=3.10"
+
+
+mdanalysis_requires: ">=2.0.0"
+
+
+run_tests:
+  - pytest --pyargs menger
+
+
+test_dependencies:
+  - mamba install pytest MDAnalysisTests
+
+
+project_org: EtienneReboul
+
+
+install:
```

Try installing numba here with mamba. This may prevent MDAnalysis from pulling in numpy ≥ 2.2. Perhaps just put all your dependencies into a mamba installation line so that when MDAnalysis gets installed as part of the kit testing, the mamba pins are in place
```
- mamba install "numba>=0.60.0" "numpy<=2.1" MDAnalysis
- pip install menger-curvature
```

---

Actually, do not include numpy here: numba properly pins numpy in their conda recipe https://github.com/conda-forge/numba-feedstock/blob/b34a0b3534b52413e43f01d3d97a3a97dac30b3d/recipe/meta.yaml#L50 

(I am not sure about their PyPi package because the [numa setup.py](https://github.com/numba/numba/blob/main/setup.py) does not seem to define a max_numpy_version so you might need to explicitly pin in your pyproject.toml ... you'll need to check.)

---

You still need to do the mamba install for numba
```suggestion
install:
  - mamba install -c conda-forge "numpy<=2.1" numba 
```
Given that the above worked for `src_install`, try the same here. 

*If* it works then I'd remove the "numpy<=2.1" and test it it still works.